guides/common/modules/proc_configuring-project-settings-for-keycloak-authentication-using-the-web-ui.adoc

```diff
@@ -0,0 +1,32 @@
+[id="configuring-{project-context}-settings-for-keycloak-authentication-using-the-web-ui_{context}"]
+= Configuring {Project} settings for {keycloak} authentication using the web UI
+
+If you are configuring a client that will provide {ProjectWebUI} authentication, configure the {Project} client and add {keycloak} as an external authentication source.
+
+.Prerequisites
+
+* For {keycloak-quarkus}: Ensure that the *Client authentication* setting in the {Project} client in the {keycloak-quarkus} web UI is enabled.
+For more information, see xref:configuring-the-{project-context}-client-in-keycloak_keycloak-quarkus[].
++
+For {keycloak-wildfly}: Ensure that the *Access Type* setting in the {Project} client in the {keycloak-wildfly} web UI is set to *confidential*.
+For more information, see xref:configuring-the-{project-context}-client-in-keycloak_keycloak-wildfly[].
+* Obtain the values to configure {Project} settings from the following URL: `https://{keycloak-example-com}/auth/realms/{Project}_Realm/.well-known/openid-configuration`
+
+.Procedure
+
+In the {ProjectWebUI}:
+
+. Navigate to *Administer* > *Settings*.
+. On the *Authentication* tab, configure the following settings:
+.. *Authorize login delegation*: Set to `Yes`.
+.. *Authorize login delegation auth source user autocreate*: Set to `External`.
+.. *Login delegation logout URL*: Set to `\https://{foreman-example-com}/users/extlogout`.
+.. *OIDC Algorithm*: Set to `RS256`.
```

Comment:
```suggestion
.. *OIDC Algorithm*: For example, set to `RS256`.
```

Setting it to `RS256` is actually just an example, you could have totally configured something different in Keycloak, there might be a better phrasing though.